bazel/curl.BUILD

```diff
@@ -46,7 +46,9 @@ CURL_WIN_SRCS = [
     "lib/inet_ntop.c",
     "lib/system_win32.c",
     "lib/vtls/schannel.c",
+    "lib/vtls/schannel_verify.c",
     "lib/idn_win32.c",
+    "lib/x509asn1.c",
```

Comment:
@gunan JFYI, I needed to add 2 files to our curl.BUILD which is originally copied from tensorflow repo.
We had errors described in: https://github.com/googleapis/google-cloud-cpp/issues/666#issuecomment-518859170